packages/create-expo-module/src/create-expo-module.ts

```diff
@@ -3,21 +3,27 @@ import chalk from 'chalk';
 import { Command } from 'commander';
 import downloadTarball from 'download-tarball';
 import ejs from 'ejs';
+import { sync } from 'find-up';
```

Comment:
`sync` tells nothing about what the function does.
```suggestion
import findUp from 'find-up';
```